### ISSUE TITLE
Modernizr npm published package points to wrong lodash

### ISSUE THREAD

**Issue by yleviel-notarize** [Modernizr/Modernizr#2394]
Similar to #2313, retire-js notes that lodash 4.17.4 has a vulnerability.  I was about to submit a PR to update to lodash 4.17.11, and it seems this was already done in the package.json but not submitted as a new version to npm.  Can you publish a 3.6.1 with the latest so the following errors are not generated? 

```
node_modules/modernizr/node_modules/lodash/package.json
 ↳ lodash 4.17.4
lodash 4.17.4 has known vulnerabilities: severity: low; summary: Prototype pollution attack; https://hackerone.com/reports/380873
```

**Comment by rejas** [comment]
3.6.1 from master would be a little more than just this npm update. i'd love to get v3.7 though @patrickkettner

**Comment by owensgit** [comment]
Unfortunately, this is preventing me from using Modernizr at my company. Is there a timescale for a this to be published?

**Comment by rejas** [comment]
Sorry, cant help you as long as @patrickkettner is not resurfacing and getting the releases back on track. Neither do I have publishing rights, nor access to website etc which would need to be updated :-(

**Comment by wraithgar** [comment]
There are 5 other people who have access to publish this on npm.  I don't want to @ spam a bunch of people here but also am not sure what the correct protocol is on an issue like this.

According to `npm info modernizr` these people have npm publish rights to this package.  (These are npm usernames)

```
maintainers:
- kurafire
- patrickkettner
- paulirish
- ryanseddon
- slexaxton
- stucox
```

**Comment by rejas** [comment]
Not sure if @ ping would trigger a reaction of half of those from the list. Looking at their github timeline some havent done anything the last year, some at least show up, some half better things to do. Lets just hope patrickkettner comes back during the holidays for some cleanups....

**Comment by thomstak** [comment]
Same issue here, hope it will be fixed soon

**Comment by huanghai21** [comment]
hi, @patrickkettner, we got the same issue. Hope it can be published soon to resolve the `prototype pollution` vulnerability came from the npm audit.
thx~

**Comment by rejas** [comment]
Hi @all I published the current master under this package https://www.npmjs.com/package/modernizr-next if you are willing to try that out ;-)

**Comment by MarioBinder** [comment]
> Hi @ALL I published the current master under this package https://www.npmjs.com/package/modernizr-next if you are willing to try that out ;-)

The dependency underscorce.string is also affected https://www.npmjs.com/advisories/745

**Comment by rejas** [comment]
yeah, not sure if we can get that fixed easily, since we dont use it directly... Any ideas @MarioBinder ?

**Comment by rejas** [comment]
Released v3.7.0, wil be on npm in a short while
